app/config/locale/translations/en.json

```
@@ -11,13 +11,20 @@
     "emails.verification.signature": "{{project}} team",
     "emails.magicSession.subject": "{{project}} Login",
     "emails.magicSession.hello": "Hello,",
-    "emails.magicSession.optionButton": "Click the button below to securely sign in to your {{project}} account. It will expire in 1 hour.",
+    "emails.magicSession.optionButton": "Click the button below to securely sign in to your {{project}} account. This link is valid for 1 hour.",
```

```suggestion
    "emails.magicSession.optionButton": "Click the button below to securely sign in to your {{project}} account. This link will expire in 1 hour.",
```
Apologies for the double work.. It needs to be consistent with the message we use for otps -> `This code will expire in 15 minutes.`